src/shelly_hap_rgb.cpp

```diff
@@ -326,7 +342,17 @@ Status RGB::SetState(const std::string &state_json) {
   return Status::OK();
 }
 
-void RGB::AutoOffTimerCB() {
+RGBWLight::LightMode RGBWLight::getLightMode() const {
+  switch (mgos_sys_config_get_shelly_mode()) {
```

Comment:
components should not be referencing sys config directly, this breaks the abstraction boundaries.